Gemfile.lock

```diff
@@ -1,24 +1,24 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activemodel (6.1.7.1)
-      activesupport (= 6.1.7.1)
-    activerecord (6.1.7.1)
-      activemodel (= 6.1.7.1)
-      activesupport (= 6.1.7.1)
-    activesupport (6.1.7.1)
+    activemodel (6.1.7.3)
```

Comment:
Are all the gem updates necessary for Ruby 3.1 to work? 
Updating dry-configurable, sinatra and grape all at the same time feels risky.

Reply:
Grape definitely needed updating in order for the rspec tests to pass.

Reply:
If you replace Gemfile.lock with the one from the main branch and then re-run `bundle install` that should leave you with just the necessary changes